apps/site/lib/site_web/controllers/fare/commuter.ex

```diff
@@ -45,4 +49,7 @@ defmodule SiteWeb.FareController.Commuter do
   @spec foxboro?(String.t(), String.t()) :: boolean()
   defp foxboro?(a, b) when @foxboro in [a, b], do: true
   defp foxboro?(_, _), do: false
+
+  @spec foxboro_pilot?(Calendar.date()) :: boolean
```

Comment:
I see why you asked the question on the other PR. I think technically either is OK. I don't see this type in use elsewhere in our codebase but Date.compare would work with it. We expect that this date object is actually a date like ~D[2019-10-21] not a datetime or any other type since we assign it in https://github.com/mbta/dotcom/blob/master/apps/site/lib/site_web/router.ex#L23 as Timex.to_date or Util.service_date